packages/location-field/src/index.js

```diff
@@ -308,17 +318,71 @@ class LocationField extends Component {
       });
   }
 
-  render() {
+  renderFeature = (itemIndex, feature) => {
     const {
+      layerColorMap,
+      geocoderConfig,
       addLocationSearch,
+      GeocodedOptionIconComponent,
+      operatorIconMap
+    } = this.props;
+    const { activeIndex } = this.state;
+    // Create the selection handler
+    const locationSelected = () => {
+      getGeocoder(geocoderConfig)
+        .getLocationFromGeocodedFeature(feature)
+        .then(geocodedLocation => {
+          // Set the current location
+          this.setLocation(geocodedLocation, "GEOCODE");
+          // Add to the location search history. This is intended to
+          // populate the sessionSearches array.
+          addLocationSearch({ location: geocodedLocation });
+        });
+    };
+
+    // Add to the selection handler lookup (for use in onKeyDown)
+    this.locationSelectedLookup[itemIndex] = locationSelected;
+
+    // Extract GTFS/POI info and assign to class
+    const { layer, source, id } = feature.properties;
+    const classNames = [];
+    let operatorIcon;
+    // Operator only exists on transit features
+    const featureIdComponents = source === "transit" && id.split("::");
+    if (featureIdComponents.length > 0) {
+      const operatorName = featureIdComponents[1]
+        .replaceAll(" ", "-")
+        .toLowerCase();
+      classNames.push(`operator-${operatorName}`);
+      operatorIcon = operatorIconMap[operatorName];
+    }
+
+    classNames.push(`source-${source}`);
+    classNames.push(`layer-${layer}`);
+
+    // Create and return the option menu item
+    return (
+      <Option
+        icon={operatorIcon || <GeocodedOptionIconComponent feature={feature} />}
+        key={optionKey++}
+        title={feature.properties.label}
+        onClick={locationSelected}
+        isActive={itemIndex === activeIndex}
+        classes={classNames.join(" ")}
+        color={layerColorMap[layer]}
```

Comment:
Sort props

Reply:
Sorted in e0fc8a811358ed8a55a587f948db1614da68dbd4